docs/usage/Installations.md

```diff
@@ -0,0 +1,459 @@
+```
+
+### Target Exports
+
+Target exports are defined by a unique logical name and a reference to the actual target.
+
+The logical name must be unique within all exports (data and targets) and must match the export name in the blueprint.<br>
+The `target` attribute defines the name of contextified target that is created within the same namespace.
+
+```yaml
+exports:
+  targets:
+  - name: my-target
+    target: "my-exported-target"
+```
+
+will result in
+```yaml
+apiVersion: landscaper.gardener.cloud/v1alpha1
+kind: Target
+metadata:
+  name: <some hash>
+  labels:
+    data.landscaper.gardener.cloud/context: Installation.<namespace>.<installation name>
+    data.landscaper.gardener.cloud/key: my-exported-target
+    data.landscaper.gardener.cloud/source: Installation.<namespace>.<installation name>
+    data.landscaper.gardener.cloud/sourceType: export
+spec:
+  type: my-type
+  config: <exported target data>
+```
+
+### Export Data Mappings
+
+It can happen that exported data is of a different format than what is needed in the context.<br>
+One possible solution is to add an additional blueprint that transforms the data.
+As this approach would result in a big overhead for just transforming some data, an additional method is needed to transform that data.
+
+This transformation can be done in `spec.exportDataMappings`.
+`ExportDataMappings` define a map of all exports of a installation that can be templated using [spiff](https://github.com/mandelsoft/spiff). <br>
+These mappings make it possible to 
+- create more exports from one or multiple exports of a blueprint
+- combine multiple exports to one
+- export hard coded values
+
+All exported values can be accessed in the templating by their logical internal names.
+All blueprint export data mappings are optional, by default the logical internal name is matched to the blueprint export.
+```yaml
+spec:
+  exports:
+    data:
+    - name: imp1
+    - name: imp2
+  exportDataMappings:
+    imp1: hardcoded value
+    imp2: (( exp1.subkey ))
+```
+
+__Example__
+```yaml
+apiVersion: landscaper.gardener.cloud/v1alpha1
+kind: Blueprint
+exports:
+- name: identifier
+  schema:
+    type: string
+- name: aws-credentials
+  schema:
+    type: object
+    properties:
+      accessKeyID:
+        type: string
+      accessKeySecret:
+        type: string
+- name: gcp-credentials
+  schema:
+    type: object
+    properties:
+      serviceaccount.yaml:
+        type: string
+```
+
+```yaml
+spec:
+  exports:
+    data:
+    - name: identifier
+    - name: creds 
+  exportDataMappings:
+    identifier: (( identifier ))
+    creds:
+    - type: aws
+      creds: (( aws-credentials ))
+    - type: gcp
+      creds: (( gcp-credentials ))
+```
+
+## Operations
+
+The behavior of a installation is set by using operation annotations.
+These annotations are either automatically set by the landscaper itself as part of the dafault reconiclition loop.
+Or operator can set annoations to force specific behavior.
```

Comment:
`either` in one sentence and `or` in the next one sounds strange to me. And there is a typo in `reconciliation`.
Proposal 1:
```
These annotations are set automatically by the landscaper as part of the default reconciliation loop.
An operator can also set annotations manually to enforce a specific behaviour.
```
(BE vs AE - behavior or behaviour?)

Proposal 2:
```
These annotations are either set automatically by the landscaper itself as part of the default reconciliation loop, or they can be set manually by an operator to enforce a specific behaviour.
```

Reply:
I think we should stay with one form. therefore will go with AE